docs/tutorials/uniq-factories/uniq-offer/uniq-offer.experimental.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Uniq Offer'
+
+order: 1
+---
+
+# Uniq Offer
+
+## Overview of Uniq Offer feature
+
+Uniq Offer allow user to make a buy offer directly to the owner of Uniq or Uniq Factory if it's tradable.
+
+Once these offers are made, owner of Uniq or Uniq Factory can accept these offers as long as offer is still valid. This means offers remain valid even if the Uniq changes hands, ensuring your offer stands until it expires or is withdrawn.
```

Comment:
Should be
`Once these offers are made, the owner of Uniq can accept these offers as long as they are valid.`

Reply:
fixed